lib/icinga/downtime.cpp

```diff
@@ -557,7 +565,7 @@ void Downtime::DowntimesOrphanedTimerHandler()
 	for (const Downtime::Ptr& downtime : ConfigType::GetObjectsByType<Downtime>()) {
 		/* Only remove downtimes which are activated after daemon start. */
 		if (downtime->IsActive() && !downtime->HasValidConfigOwner())
-			RemoveDowntime(downtime->GetName(), false, false, true);
+			RemoveDowntime(downtime->GetName(), false, DowntimeRemovedForce);
```

Comment:
Here the config owner is the reason.

Reply:
That's not the only reason here!
https://github.com/Icinga/icinga2/blob/252b075d0f71013d37dd088fb17b58999ca3ef78/lib/icinga/downtime.cpp#L237-L242